aggregate/housing_production/area_within_historic_district.py

```diff
@@ -22,6 +23,11 @@ def area_historic_internal_review():
         "housing_production",
     )
 
+def rename_col(cols) -> List:
```

Comment:
this is fine but it introduces a lot of complexity for two column labels to be changed. In my solution I just changed the lines where the new columns were set